src/renderer/modals/DisclaimerModal/index.js

```diff
@@ -114,7 +114,7 @@ class DisclaimerModal extends PureComponent<Props, State> {
             <Box horizontal justifyContent="flex-end" alignItems="center" style={{ flex: 1 }}>
               <Box horizontal alignItems="center" onClick={this.onSeedReady} style={{ flex: 1 }}>
                 <CheckBox
-                  id={"firmware-update-disclaimer-modal-seed-ready-checkbox"}
```

Comment:
Are you sure this ID isn't getting used for anything else (like styling)?

Reply:
as a convention, we try to not use IDs for styling, but nice catch anyway, we should indeed check it out (even tho i'm 99% sure we don't need to)

Reply:
Sure, I looked for the id in the whole code.